### DISCUSSION TITLE
Should the latest preview of Shiny.Net.Http Work?

### DISCUSSION THREAD

**Discussion by JoacimWall** [shinyorg/shiny#1203] [Q&A]
Hi
I try the latest Preview and I add a download to the Queue. 
I see the status is completed but there is no file saved in the path I provide.   
and the events OnCompleted, OnError  in the IHttpTransferDelegate is newer trigged.
On iOS the contractor of IHttpTransferDelegate is never run but it does on Android.

Feels like it's I missing something.

I have set 
.UseMauiApp<App>()
and the 
builder.Services.AddHttpTransfers<SampleHttpTransferDelegate>();

//Joacim

//Regards

**Comment by aritchie** [comment]
It's an alpha, so nothing is guaranteed to run which is why I'm not supporting the alphas at this time!
